src/renate/updaters/learner.py

```diff
@@ -224,11 +182,17 @@ def forward(self, inputs: NestedTensors, task_id: Optional[str] = None) -> torch
             task_id = self._task_id
         return self._model(inputs, task_id=task_id)
 
+    def training_step_unpack_batch(
+        self, batch: Tuple[NestedTensors, torch.Tensor]
+    ) -> Tuple[NestedTensors, Any]:
+        inputs, targets = batch
```

Comment:
This choice of what a batch comprises is specific to HF vs non-HF datasets? So, why is this specified in the base class?